modules/programs/fff.nix

```diff
@@ -0,0 +1,582 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.fff;
+
+in {
+  meta.maintainers = [ maintainers.gigahawk ];
+
+  options.programs.fff = {
+    enable = mkEnableOption "A simple file manager written in bash";
```

Comment:
Make sure the description looks ok in manual.

Reply:
Sorry if this is a stupid question, but how do I build the manual?
I can't seem to find any documentation on how to build it.

Reply:
Either
```console
$ man home-configuration.nix
```
for the man page or add `manual.html.enable = true` to your configuration and run
```console
$ home-manager-help
```
for the HTML version.